packages/create-flex-plugin/templates/js/src/components/CustomTaskListComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Div, Accented } from './styled';
+
+// It is recommended to keep components stateless and use redux for managing states
+const CustomTaskListComponent = (props) => {
+  if (!props.isOpen) {
+    return null;
+  }
+
+  return (
```

Comment:
or just
```
return Boolean(props.isOpen) &&  (
  <Div>
  ...
```

Reply:
I prefer the first one; it's cleaner IMO. I use that notation if this is inline conditional rendering.

Reply:
Sure np. All of these are just comments, no change requests